apps/website/src/routes/docs/headless/breadcrumb/BreadcrumbWrapper.tsx

```diff
@@ -0,0 +1,12 @@
+import { component$, Slot } from '@builder.io/qwik';
+
+export default component$((props: { title?: string; class?: string }) => {
+  return (
+    <div class={props.class || ''}>
+      {props.title && <h1 class="mt-8 mb-4">{props.title}</h1>}
+      <div class="dark:bg-slate-900 bg-slate-200 rounded p-3">
```

Comment:
Headless documentation should not contain daisy classes. You have to create a custom CSS file for that. Thanks

Reply:
Ah, I forgot about that 😢. I'm going to PR again ASAP

Reply:
No problem. Thanks for your commitment 😉

Reply:
Already updated